tests/app/tests/test_webpack.py

```diff
@@ -26,6 +27,10 @@
 class LoaderTestCase(TestCase):
     def setUp(self):
         self.factory = RequestFactory()
+        self.cleanup_bundles_folder()
+
+    def cleanup_bundles_folder(self):
+        rmtree('./assets/bundles', ignore_errors=True)
```

Comment:
Not sure if this is really safe. I imagine some crazy situation of this test running in someones app (locally, but still).

Maybe the test setting should set a more custom name for the bundles dir?

Reply:
Done